graphene_sqlalchemy/registry.py

```diff
@@ -27,6 +88,34 @@ def register_composite_converter(self, composite, converter):
     def get_converter_for_composite(self, composite):
         return self._registry_composites.get(composite)
 
+    def get_type_for_enum(self, sql_type):
+        if not isinstance(sql_type, SQLAlchemyEnumType):
+            raise TypeError(
+                "Only sqlalchemy.Enum objects can be registered as enum, "
+                'received "{}"'.format(sql_type)
+            )
+        enum_class = sql_type.enum_class
+        if enum_class:
+            name = enum_class.__name__
+            members = OrderedDict(
+                (to_enum_value_name(key), value.value)
+                for key, value in enum_class.__members__.items()
+            )
+        else:
+            name = sql_type.name
+            name = (
+                to_type_name(name)
+                if name
+                else "Enum{}".format(len(self._registry_enums) + 1)
```

Comment:
This is the name of the SQLAlchemy Enum type, not the name of the column. That name may not exist.

Reply:
You would need to get the SQLAlchemy enum type anyway with `column.type`. The `convert_sqlalchemy_column` function already does this. We need the name of the enum type, not the name of the column, because the same enum could be used in different columns.